lib/platform/gitlab/index.spec.ts

```diff
@@ -1352,10 +1352,7 @@ describe(getName(), () => {
         prTitle: 'some-title',
         prBody: 'the-body',
         labels: [],
-        platformOptions: {
-          azureAutoComplete: false,
-          gitLabAutomerge: true,
-        },
+        platformOptions: { usePlatformAutomerge: true },
```

Comment:
Is this now a platform option though, if it's generic?

Reply:
For me, it seems that type of `platformOptions` (i.e. `PlatformPrOptions`) is just a tiny `RenovateConfig`. If I'm correct, it's meant to be used as configurator for `createPr()` and `updatePr()` execution. As an alternative, we can pass `automerge` and `automergeType`, though everything other than `pr-auto` would be irrelevant there.

Reply:
I'm not sure platformOptions is applicable now that we've made it a generic option. @viceice @JamieMagee wdyt?